e2e/test/scenarios/admin/database-connection-strings.cy.spec.ts

```diff
@@ -0,0 +1,240 @@
+beforeEach(() => {
+  cy.H.restore();
+  cy.signInAsAdmin();
+});
+
+function selectFieldOption(fieldName: string, option: string) {
+  cy.findByLabelText(fieldName).click();
+  cy.H.popover().contains(option).click({ force: true });
+}
+
+function chooseDatabase(database: string) {
+  selectFieldOption("Database type", database);
+}
+
+const databaseTestCases = [
```

Comment:
I think the unit test coverage on these is pretty good actually. instead of testing all of these here, let's just test postgres + mysql, but also actually test them end-to-end so that we can actually connect to the QA dbs. I'm pretty sure testing the actual connection would fail in this branch's current state due to the engine selection bug